README.md

```diff
@@ -84,6 +84,8 @@ To install this gem onto your local machine, run `bundle exec rake install`. To
 
 ## Contributing
 
+IMPORTANT: Please use [RuboCop](https://github.com/bbatsov/rubocop) and [Overcommit](https://github.com/brigade/overcommit) when submitting pull requests.
```

Comment:
Hmm.. Haven't tried that before. Added https://github.com/gsmendoza/test_changes/issues/6.